backend/apps/ai/management/commands/ai_create_project_chunks.py

```
@@ -0,0 +1,234 @@
+from django.core.management.base import BaseCommand
+
+from apps.ai.common.constants import (
+    DEFAULT_LAST_REQUEST_OFFSET_SECONDS,
+    DELIMITER,
+    MIN_REQUEST_INTERVAL_SECONDS,
+)
+from apps.ai.models.chunk import Chunk
+from apps.owasp.models.project import Project
+
+
+class Command(BaseCommand):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)